### DISCUSSION TITLE
How to share public dir with Windows guest?

### DISCUSSION THREAD

**Discussion by NestorLiao** [quickemu-project/quickemu#975] [Quickemu]
Using nixos, run quickemu  --public-dir /home/nestor/  --vm windows-11.conf --display spice, but no new driver appears. Do quickemu support it?

**Comment by lj3954** [comment]
Under the Samba tab of the README, instructions are included.

**Comment by NestorLiao** [comment]
I installed samba,  followed [this issue](https://github.com/quickemu-project/quickemu/issues/722), 
sudo ln -sf /nix/store/05g445bpar2qvzkipgpf9ba56pfqy2d0-samba-4.17.7/bin/smbd /usr/sbin/smbd
then  quickemu  --public-dir /home/nestor/  --vm windows-11.conf --display spice
got print  - smbd:     On guest: smb://10.0.2.4/qemu
but in windows guest add network location, add \\10.0.2.4\qemu got not responding and shows not appear vaild.
In hosts terminal shows: qemu-system-x86_64: Slirp: fork_exec: Failed to execute child process “/usr/sbin/smbd” (Permission denied), tried to chown/chmod, but not work.
